Adyen/HttpClient/HttpClientWrapper.cs

```diff
@@ -0,0 +1,148 @@
+#region License
+
+// /*
+//  *                       ######
+//  *                       ######
+//  * ############    ####( ######  #####. ######  ############   ############
+//  * #############  #####( ######  #####. ######  #############  #############
+//  *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  * ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  * #############  #############  #############  #############  #####  ######
+//  *  ############   ############  #############   ############  #####  ######
+//  *                                      ######
+//  *                               #############
+//  *                               ############
+//  *
+//  * Adyen Dotnet API Library
+//  *
+//  * Copyright (c) 2021 Adyen B.V.
+//  * This file is open source and available under the MIT license.
+//  * See the LICENSE file for more info.
+//  */
+
+#endregion
+
+using Adyen.Constants;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Adyen.HttpClient.Interfaces;
+using Adyen.Model;
+
+namespace Adyen.HttpClient
```

Comment:
The `httpclient` update should be done in one pull request as it is [here](https://github.com/Adyen/adyen-dotnet-api-library/pull/690) and not in this one. This one should be only for LegalEntityManagement endpoints

Reply:
Yep noted, however I did need the Restful client for these endpoints. Hence doing it this way and asking to approve the previous pr's (that include the client and automation script) first :)